### ISSUE TITLE
[Bug] Not able to properly install cosma

### ISSUE THREAD

**Issue by mefor44** [graphlab-fr/cosma#147]
### What version of Cosma are you using?

Newest version (installed via `npm install @graphlab-fr/cosma -g`)

### On which operating system(s) did you observe the bug?

- [ ] macOS
- [X] Windows
- [ ] Linux

### What version of the operating system are you on?

Windows 10

### Actions leading to the bug

I followed steps from [installation guide](https://cosma.arthurperret.fr/installing.html )

### Bug description

After I install `cosma`, when I try to run `cosma -v` I get following error:
```
node:internal/url:1343
    throw new ERR_INVALID_FILE_URL_PATH('must be absolute');
    ^

TypeError [ERR_INVALID_FILE_URL_PATH]: File URL path must be absolute
    at new NodeError (node:internal/errors:371:5)
    at getPathFromURLWin32 (node:internal/url:1343:11)
    at fileURLToPath (node:internal/url:1373:22)
    at C:\Users\marzecm\node_modules\@graphlab-fr\cosma\dist\back.cjs:3:705569
    at C:\Users\marzecm\node_modules\@graphlab-fr\cosma\dist\back.cjs:3:1288359
    at Object.<anonymous> (C:\Users\marzecm\node_modules\@graphlab-fr\cosma\dist\back.cjs:3:1288363)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12) {
  code: 'ERR_INVALID_FILE_URL_PATH'
}```

### Additional information

_No response_

**Comment by infologie** [comment]
@mefor44 Fixed by @Myllaume in [v2.5.4](https://github.com/graphlab-fr/cosma/releases/tag/2.5.4).

**Comment by mefor44** [comment]
Thanks :)
